app/models/access_token.rb

```diff
@@ -92,10 +96,11 @@ def self.allowed_scopes
   validates :permission, inclusion: { in: PERMISSIONS.values }, length: { maximum: 255 }
   validates :scopes, length: { minimum: 1, maximum: 65535 }
   validate :validate_scope_exists
+  validate :validate_expiration_date
```

Comment:
```suggestion
  validate :validate_expiration_date, on: %i[create update]
```

Reply:
I actually wish we could use Rails 7 validation like... next month maybe :lady_beetle: 
```suggestion
  validates :expires_at, comparison: { greater_than: ->{ Time.now } }, allow_blank: true
```

Reply:
Updating a token shouldn't allow editing the expiration time, so only on `:create` is needed https://github.com/3scale/porta/pull/3939/commits/df3a218aa828f46ca8f3aab6cffbc10296ecd4da

Reply:
One never knows but fine with me.